### PR TITLE
Added seed data for user.

### LINKED ISSUE #102: Add seed data for user
Add seed data for user

### PR DESCRIPTION
**Pull request by sudeeptarlekar**
This fixes #102 

Added seed data for user
username : `tony@example.com`
password : `welcome`

@vipulnsward _a please review

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -7,40 +7,44 @@
...
+ArticleUrl.find_or_create_by! article_id: a7.id, url_id: u1.id
+
+ArticleUrl.find_or_create_by! article_id: a2.id, url_id: u5.id
+ArticleUrl.find_or_create_by! article_id: a4.id, url_id: u5.id
+ArticleUrl.find_or_create_by! article_id: a6.id, url_id: u6.id
+ArticleUrl.find_or_create_by! article_id: a2.id, url_id: u7.id
+ArticleUrl.find_or_create_by! article_id: a4.id, url_id: u6.id
+ArticleUrl.find_or_create_by! article_id: a6.id, url_id: u7.id
+
+user1 = User.find_or_initialize_by email: "tony@example.com", first_name: "Tony", last_name: "Stark"
```
Comment (vipulnsward) [inline db/seeds.rb]: Please change to sam@example.com/Sam Smith, avoid movie names

File: db/seeds.rb [hunk 2]
```diff
@@ -7,40 +7,44 @@
...
-ArticleUrl.create! article_id: a3.id, url_id: u2.id
-ArticleUrl.create! article_id: a5.id, url_id: u2.id
-ArticleUrl.create! article_id: a7.id, url_id: u1.id
-
-ArticleUrl.create! article_id: a2.id, url_id: u5.id
-ArticleUrl.create! article_id: a4.id, url_id: u5.id
-ArticleUrl.create! article_id: a6.id, url_id: u6.id
-ArticleUrl.create! article_id: a2.id, url_id: u7.id
-ArticleUrl.create! article_id: a4.id, url_id: u6.id
-ArticleUrl.create! article_id: a6.id, url_id: u7.id
```
Comment (vipulnsward) [inline db/seeds.rb]: Hmm, running seed to should probably kill all existing data.

Can you see how wheel does this? 

Also no need for find_or_create in that case

Comment (sudeeptarlekar) [reply db/seeds.rb]: Wheel does not have seed data, though in `AceInvoice-Web` we are deleting all data.

File: db/seeds.rb [hunk 3]
```diff
@@ -7,6 +7,14 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+ArticleUrl.delete_all
+Category.delete_all
+Url.delete_all
+Article.delete_all
+Organization.delete_all
+User.delete_all
```
Comment (vipulnsward) [inline db/seeds.rb]: See https://github.com/bigbinary/wheel/blob/master/lib/tasks/setup.rake on how to delete/setup

**Comment by vipulnsward** [comment]
@sudeeptarlekar _a

**Pull request merged**